### PR TITLE
Adjust Travis config to support SS3

### PR DESCRIPTION
**Pull request by Cheddam**
Tests will now run! (They still fail, but one challenge at a time.)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,35 +25,21 @@ before_script:
...
 
   # Install NPM dependencies
   - if [[ $NPM_TEST ]]; then nvm install $TRAVIS_NODE_VERSION && nvm use $TRAVIS_NODE_VERSION && npm install -g yarn && yarn install --network-concurrency 1 && yarn run build; fi
 
 script:
-  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit; fi
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml tests; fi
-  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs src/ tests/; fi
+  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit mfa/tests; fi
+  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs --standard=mfa/phpcs.xml.dist login-forms/src mfa/tests; fi
```
Comment (robbieaverill) [inline .travis.yml]: `mfa/src`?

File: .travis.yml [hunk 2]
```diff
@@ -37,35 +25,21 @@ before_script:
...
   - if [[ $NPM_TEST ]]; then git diff-files --quiet -w --relative=client; fi
   - if [[ $NPM_TEST ]]; then git diff --name-status --relative=client; fi
   - if [[ $NPM_TEST ]]; then yarn run coverage; fi
   - if [[ $NPM_TEST ]]; then yarn run lint; fi
```
Comment (robbieaverill) [inline .travis.yml]: These commands will need to be run inside `mfa` folder context, e.g. `]]; then (cd mfa && yarn run lint); fi`

File: .travis.yml [hunk 3]
```diff
@@ -37,35 +25,21 @@ before_script:
...
-  - if [[ $BEHAT_TEST ]]; then sh -e /etc/init.d/xvfb start; sleep 3; fi
-  - if [[ $BEHAT_TEST ]]; then (chromedriver > artifacts/chromedriver.log 2>&1 &); fi
-  - if [[ $BEHAT_TEST ]]; then (vendor/bin/serve --bootstrap-file vendor/silverstripe/cms/tests/behat/serve-bootstrap.php &> artifacts/serve.log &); fi
+  # Dependencies
+  - git clone git://github.com/silverstripe/silverstripe-travis-support.git ~/travis-support
+  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
+  - cd ~/builds/ss
 
   # Install NPM dependencies
   - if [[ $NPM_TEST ]]; then nvm install $TRAVIS_NODE_VERSION && nvm use $TRAVIS_NODE_VERSION && npm install -g yarn && yarn install --network-concurrency 1 && yarn run build; fi
```
Comment (robbieaverill) [inline .travis.yml]: As below, you'll need to run this in brackets with a `cd mfa`

**Comment by Cheddam** [comment]
@robbieaverill I've updated the config per your feedback. The builds are still failing / erroring, but with expected output.

**Pull request merged**